system/ee/ExpressionEngine/Addons/channel/libraries/channel_form/Channel_form_lib.php

```diff
@@ -2826,8 +2825,31 @@ public function initialize($reinitialize = false)
 
         $this->fetch_settings();
 
-        $this->option_fields = $this->native_option_fields;
+        if (empty($this->option_fields)) {
+
+            $cache_key = '/Fieldtype/OptionFieldtypes';
+            $this->option_fields = ee()->cache->get($cache_key);
+
+            if (empty($this->option_fields)) {
+                $fieldtypes = ee('Model')->get('Fieldtype')->all()->pluck('name');
+                ee()->legacy_api->instantiate('channel_fields');
```

Comment:
I think you might need to add `ee()->load->library('api');` call as this function seems to be executed before API library is loaded.

Reply:
Line added. However, for our instance it was working as is.